# oppOntology
Omics Pilot Platform of Ontology (oppOntology) is a kind of enrichment analysis  tool for omics data with a graphical user interface (GUI), founded on the architecture  of MATLAB AppDesigner. oppOntology supports the enrichment calculation of GO (Gene Ontology), KEGG (Kyoto Encyclopedia of Genes and Genomes), HPO (Human  Phenotype Ontology), COG (Clusters of Orthologous Groups), MsigDB (Molecular  Signatures Database) and Function (custom functions). Enrichment contraposes function categories of specific genes which contains the calculation of count, gene ratio,  enrich factor, hypergeometry test, Fisher test, etc. This oppOntology supports the  enrichment degree calculation of multiple samples simultaneously, and diagraming  bubble graphs of all samples and various scores. The databases of GO, KEGG, COG and HPO are built-in databases in oppOntology whereas the data in database of MsigDB as the format of gmt could be downloaded in the official website of GSEA (http://www.gsea-msigdb.org/gsea/downloads.jsp). The files of Function comprised  the text files of Function ID, Function Name and Gene ID, of which the users could  define the text in the files to obtain the enrichment degree of the molecular function as  their interest. oppOntology is straightforward for users and suitable for all the versions  after MATLAB r2021. In the environment of MATLAB, the oppOntology could be  started into the operation interface when the users type in the command “oppOntology”.

![Logo](https://github.com/HangZhouSheep/oppOntology/blob/main/Logo.png)

## Tool Download Link
[oppOntology.mlappinstall](https://github.com/HangZhouSheep/oppOntology/blob/main/oppOntology.mlappinstall)

## Data Link
[data](https://github.com/HangZhouSheep/oppOntology/tree/main/Data)

## Instruction Link
[instruction](https://github.com/HangZhouSheep/oppOntology/tree/main/Instruction)
